.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+name: tests
+
+on:
+  push:
+
+jobs:
+  rspec:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version: ["3.0", "3.1", "3.2", "3.3"]
```

Comment:
Are we dropping support for Ruby 2.x here?

Reply:
updated